tools/erc-repository-indexer/erc-contract-indexer/README.md

```diff
@@ -35,12 +35,14 @@ The ERC Contract Indexer is a tool designed to facilitate the indexing and manag
-| `STARTING_POINT`       | The starting point for contract indexing.                                                                    | A Hedera contract ID (e.g., `0.0.369`), an EVM 20-byte address (e.g., `0x0000000000000000000000000000000000000369`), or a get contract list next pointer (e.g., `/api/v1/contracts?limit=100&order=asc&contract.id=gt:0.0.369`) |
+| Variable               | Description                                                                                                    | Accepted Values                                                                                                                                                                                                                 |
+| ---------------------- | -------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `HEDERA_NETWORK`       | The network to connect to.                                                                                     | `local-node`, `previewnet`, `testnet`, or `mainnet`                                                                                                                                                                             |
+| `MIRROR_NODE_URL`      | The URL for the Hedera mirror node API.                                                                        | A valid URL pointing to the Hedera mirror node (e.g., `https://{previewnet\|testnet\|mainnet}.mirrornode.hedera.com`)                                                                                                           |
+| `MIRROR_NODE_URL_WEB3` | The URL for the Hedera Mirror Node Web3Module API, required only when `HEDERA_NETWORK` is set to `local-node`. | Any value                                                                                                                                                                                                                       |
+| `STARTING_POINT`       | The starting point for contract indexing.                                                                      | A Hedera contract ID (e.g., `0.0.369`), an EVM 20-byte address (e.g., `0x0000000000000000000000000000000000000369`), or a get contract list next pointer (e.g., `/api/v1/contracts?limit=100&order=asc&contract.id=gt:0.0.369`) |
+| `SDK_OPERATOR_ID`      | The operator ID used to initialize a Hashgraph SDK, applicable only for testing.                               | Hedera Account IDs                                                                                                                                                                                                              |
```

Comment:
If just for testing pull them out of this section and either put in a testing section or at least a separate table for testing

Reply:
Gotcha will update in follow up PR!